subsys/net/l2/openthread/Kconfig

```diff
@@ -162,6 +162,16 @@ config OPENTHREAD_JOINER_PSKD
 	help
 	  Pre Shared Key for the Device to start joiner
 
+config OPENTHREAD_DHCP6_CLIENT
+       bool "DHCPv6 client support"
+       help
+         Enable Openthread DHCPv6 client capability in OpenThread stack
```

Comment:
nit: IMO there's too many OpenThreads in this sentence, how about: 
`Enable DHCPv6 client capability in OpenThread stack`
Same for the server.

Reply:
Done